File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisModuleExistsException.java
package org.irislang.jiris.core.exceptions.fatal;

/**
 * Created by Huisama on 2017/5/25 0025.
 */
public class IrisModuleExistsException  extends IrisFatalException {
    public IrisModuleExistsException(String fileName, int lineNumber, String message) {
        super(fileName, lineNumber, message);
    }

    @Override
    public String GetFatalExceptionName() {
        return "ModuleExistsIrregular";
    }
}

File: src/main/java/org/irislang/jiris/core/exceptions/fatal/IrisModuleExistsException.kt
package org.irislang.jiris.core.exceptions.fatal

/**
 * Created by Huisama on 2017/5/25 0025.
 */
class IrisModuleExistsException(fileName: String, lineNumber: Int, message: String) : IrisFatalException(fileName, lineNumber, message) {

    override fun GetFatalExceptionName(): String {
        return "ModuleExistsIrregular"
    }
}
